addons/web/static/src/views/fields/relational_utils.js

```diff
@@ -350,10 +350,11 @@ export function useOpenMany2XRecord({
     const addDialog = useOwnedDialogs();
     const orm = useService("orm");
 
-    return async function openDialog({ resId = false, title, context }, immediate = false) {
+    return async function openDialog({ resId = false, newModel = null, title, context }, immediate = false) {
+        const model = newModel || resModel;
```

Comment:
Hmmm, "newModel" ?

"forceModel" maybe ?

However it seems strange to call "openDialog" on something else than the resModel defined when calling "useOpenMany2XRecord".

Not sure what this is for, but maybe we should instead completely call "useOpenMany2XRecord" and pass on the new model to handle model change?

Because I'm not sure the JS team will like this change ^^